assets/blocks/learner-courses-block/learner-courses-editor.scss

```diff
@@ -4,6 +4,20 @@
 $block: '.wp-block-sensei-lms-learner-courses';
 $courses-list: '#{$block}__courses-list';
 
+.editor-styles-wrapper #{$block} {
+	&__deprecation-notice {
+		margin-bottom: 2rem;
+
+		.components-notice__actions {
```

Comment:
Unfortunately, I had to add these styles because the actions aren't looking good in the editor.